src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -669,20 +669,22 @@ public static bool ParseDeclaredSymbolId(string id, Compilation compilation, Lis
                 }
 
                 int index = 0;
+                id = id.Replace(" ", string.Empty);
```

Comment:
This implies more iteration of the string so I am wondering if that's ok.

Reply:
I'll try to think of another approach.